english/time-travel/0/10/10-3B.md

```diff
@@ -2,7 +2,7 @@
 
 You survived and killed werewolf 🐺! Now you can play another dungeons or rooms! 
 
-- [Start from beggining](../begin-journey.md)
+- [Start from beggining](../../begin-journey.md)
```

Comment:
Can you please fix this typo : beggining to beginning?

Reply:
I am on it already! Thanks!

Reply:
Done!
If you see more words/sentences to change, I will really appreciate if you share with me! :)